charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -59,50 +59,108 @@ spec:
   selector:
     matchLabels:
       app: etcd-statefulset
-      role: {{.Values.role}}
+      role: {{ .Values.role }}
   template:
     metadata:
       labels:
         app: etcd-statefulset
-        role: {{.Values.role}}
+        role: {{ .Values.role }}
     spec:
       containers:
-      - name: etcd-container
+      - name: etcd
         image: {{ index .Values.images "etcd" }}
         imagePullPolicy: IfNotPresent
         command:
-        - "/usr/local/bin/etcd"
-        - "--data-dir=/var/etcd/data"
-        - "--name=etcd-{{.Values.role}}"
-        - "--advertise-client-urls=http://etcd-{{.Values.role}}-0.etcd-{{.Values.role}}:2379"
-        - "--listen-peer-urls=http://0.0.0.0:2380"
-        - "--listen-client-urls=http://0.0.0.0:2379"
-        - "--initial-advertise-peer-urls=http://etcd-{{.Values.role}}-0.etcd-{{.Values.role}}:2380"
-        - "--initial-cluster=etcd-{{.Values.role}}=http://etcd-{{.Values.role}}-0.etcd-{{.Values.role}}:2380"
-        - "--initial-cluster-state=new"
-        - "--initial-cluster-token=new"
+        - /bin/sh
```

Comment:
Why don't you put this entire script in ConfigMap, mount it in the container and run it?